src/app/api/modules/module-requests/[id]/route.ts

```
@@ -0,0 +1,131 @@
+import { NextRequest } from "next/server";
+import type { TablesUpdate } from "@/lib/types/db";
+import { requireAuth } from "../_auth";
+import type { ModuleRequestSpec } from "../lib";
+import type { ModuleRequestSpec } from "../lib";
```

Biome (2.3.14)

[error] Shouldn't redeclare 'ModuleRequestSpec'. Consider to delete it or rename it.

'ModuleRequestSpec' is defined here:


(lint/suspicious/noRedeclare)